src/style.css

```
@@ -1 +1,76 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+.html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  line-height: 60px;
+  position: relative;
+  margin-right: 20px;
+}
+
+.image__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link > a {
+  display: block;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+  font-size: 12px;
+}
+
+.nav__link > a:hover {
+  color: #00acdc;
+  height: 60px;
+}
```

Why do you set height on hover?
```suggestion
.nav__link > a:hover {
  color: #00acdc;
}
```